dist/angular-breadcrumb.js

```diff
@@ -212,6 +216,7 @@ function BreadcrumbDirective($interpolate, $breadcrumb, $rootScope) {
                     scope.steps = $breadcrumb.getStatesChain();
                     angular.forEach(scope.steps, function (step) {
                         if (step.ncyBreadcrumb && step.ncyBreadcrumb.label) {
+                            viewScope.ncyStateParams = $breadcrumb.getStateParams();
```

Comment:
Hi, many thanks for the PR.

I don't understand why you attach the stateParams to the `viewScope` here : it is the same scope of the one handled by the view controller (where you can already inject $stateParams and use it). 

It make more sense if you attach the stateParams to the scope of the directive (`scope`) in order to use it in custom templates. Maybe, is that you wanted to do ?